### ISSUE TITLE
Skype process remains in background after closing the app

### ISSUE THREAD

**Issue by bertogg** [flathub/com.skype.Client#135]
Hi,

I noticed that if I run Skype and then quit the app (File -> Quit) the skype process remains in the background:

```
 7410 pts/2    S      0:00 bwrap --args 37 skype
 7421 pts/2    S      0:00 /app/extra/share/skypeforlinux/chrome-sandbox /app/extra/share/skypeforlinux/skypeforlinux --type=zygote ...
```

That happens each time I run Skype, i.e. if I run it 3 times then I end up with three separate stale processes.

I have Skype 8.68.0.100, commit eedbc84efb8a57ce326c1035e3b1be9ae33e2a038b1fa72134aada271814e470 and Flatpak 1.2.5.

**Comment by AleksejShilin** [comment]
I have this issue with version 8.68 and flatpak 1.2.5 as in Debian stable, too. Neither the one installed from the official deb package nor the previous 8.67 seems to be affected.

**Comment by AleksejShilin** [comment]
I've also tested with flatpak 1.10.1, and the issue is still there. Moreover, other Electron-based apps like Slack are also affected, i.e. it is not a Skype problem per se but a wider one.

**Comment by AleksejShilin** [comment]
Fixed in version 8.71.0.36.
